src/sequelize.ts

```diff
@@ -2,29 +2,39 @@ import { Sequelize } from "sequelize";
 import type { Application } from "./declarations";
 import logger from "./logger";
 
-export function setupSequelize (app: Application): void {
+export function setupSequelize(app: Application): void {
+	const dialect = "postgres"; // Or your dialect name
```

Comment:
Can we rename this variable to something more descriptive?

Reply:
Hmm I took that from the official doc. I think.

Not sure what to name it. `queryDialect`?